test/Sentry.Unity.Editor.iOS.Tests/BuildPostProcessorTests.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using Sentry.Unity.Tests.SharedClasses;
+
+namespace Sentry.Unity.Editor.iOS.Tests
+{
+    public class BuildPostProcessorTests
+    {
+        public class Fixture
+        {
+            public string TestDirectoryPath { get; set; }
```

Comment:
i dont understand the `Fixture` pattern. why not make these fields on `BuildPostProcessorTests`?

Reply:
Very valid point. I had something else in mind with the fixture and then just stuck with it.